### PR TITLE
AG-12489 Ensure styles are added before instantiating modules

### PR DESCRIPTION
**Pull request by jacobp100**
Ensures styles are added to the dom before instantiating modules

### CODE REVIEW

File: packages/ag-charts-community/src/api/agCharts.ts [hunk 1]
```diff
@@ -225,7 +231,15 @@ class AgChartsInternal {
     }
 
     private static createChartInstance(options: ChartOptions, oldChart?: Chart): Chart {
-        const transferableResource = oldChart?.destroy({ keepTransferableResources: true });
+        let transferableResource: TransferableResources | undefined;
+        if (oldChart != null) {
+            transferableResource = oldChart.destroy({ keepTransferableResources: true });
+        } else {
+            transferableResource = {
```
Comment (alantreadway) [inline packages/ag-charts-community/src/api/agCharts.ts]: Why do we need to have this case now?

File: packages/ag-charts-community/src/chart/toolbar/toolbar.ts [hunk 2]
```diff
@@ -755,9 +755,9 @@ export class Toolbar extends BaseModuleInstance implements ModuleInstance {
         }
         this.updateButton(button, options);
 
-        setTimeout(() => {
+        requestAnimationFrame(() => {
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/toolbar/toolbar.ts:758]: Side-note: We need to handle these cases more elegantly, but agree `requestAnimationFrame()` is probably closer to the target than `setTimeout()`.

Comment (jacobp100) [reply packages/ag-charts-community/src/chart/toolbar/toolbar.ts:758]: We need to ensure the styles were applied, and setTimeout might be called before that

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**